src/CMakeLists.txt

```diff
@@ -41,12 +41,16 @@ if(CPP_STARTER_USE_NANA)
   add_subdirectory(nana)
 endif()
 
+find_package(fmt)
+find_package(spdlog)
+find_package(docopt)
+
 # Generic test that uses conan libs
 add_executable(intro main.cpp)
 target_link_libraries(
   intro
   PRIVATE project_options
           project_warnings
-          CONAN_PKG::docopt.cpp
-          CONAN_PKG::fmt
-          CONAN_PKG::spdlog)
+          docopt::docopt
+          fmt::fmt
+          spdlog::spdlog)
```

Comment:
This looks good. Keep in mind, we need to give the same treatment to the CMakeLists.txt files in the imgui and sdl examples as well. I did this in another branch that tried to tackle this issue here: https://github.com/ddalcino/cpp_starter_project/commit/edd75c7938b5aa9e94b52e7f3ceb10ad26455c30. I did have this working at one point, but I'm not certain it still works now.

Reply:
Oh, sorry for stepping on work you had already done!

Reply:
No problem! Your attempt is better anyway, since it's up to date on the whole Conanfile situation. I never made a PR out of this, so it's my own fault. I mentioned it in the discussion about #136 and then forgot about it.